Web/stockdogweb/src/components/Listing/Listing.scss

```diff
@@ -0,0 +1,73 @@
+@import '../../css/Global.scss';
+
+$grey-color: #afafaf;
+
+.Listing {
+   font-family: 'Assistant';
```

Comment:
Do you need to specify the font-family if you had the * rule in global.scss already?

Reply:
Yup! I do it because I'm designing these components as standalone components. Theoretically, you could use these in a totally irrelevant project and it will still look exactly the same.

Reply:
Ok 👍